.github/workflows/deploy-preview.yml

```diff
@@ -0,0 +1,50 @@
+name: 'Preview Deployment'
+on:
+  pull_request_target:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    permissions:
+      actions: read
+      contents: read
+      deployments: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Cache node_modules
+        uses: actions/cache@v4
+        with:
+          path: node_modules/
+          key: node_modules-${{ runner.os }}-${{ hashFiles('yarn.lock') }}
+
+      - name: Install node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: '.nvmrc'
+
+      - run: npm install yarn -g
+      - run: yarn install
+      - run: yarn build
+
+      - name: Deploy to Cloudflare
+        id: deploy
+        uses: cloudflare/wrangler-action@v3
+        with:
+          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          accountId: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+          command: pages deploy src/dist --project-name=${{ vars.CLOUDFLARE_PROJECT_NAME }} --commit-dirty=true
+
+      - name: Add deployment comment
+        uses: thollander/actions-comment-pull-request@v3
+        with:
+          message: |
+             Preview URL: ${{ steps.deploy.outputs.deployment-url }}
+          reactions: eyes, rocket
```

Comment:
```suggestion
```

Reply:
@blckmn - this was requested by @nerdCopter 

See https://github.com/betaflight/betaflight-configurator/pull/4455#issuecomment-2847185487

Reply:
Yes this updates the single comment (recreation mode) issue.

I'm keeping the emojis though :)